### PR TITLE
Feature/coco retheme

### PR DESCRIPTION
**Pull request by DuongTyler**
Creates the base base theme. Cards are not themed yet, and almost all the messages and titles are placeholders for the actual stuff. Will be changing a lot from here but I thought it would be a good idea if we make this master so that people don't accidentally work off the wrong branch

### CODE REVIEW

**Comment by DuongTyler** [comment]
image

File: frontend/src/pages/home/home.tsx [hunk 1]
```diff
@@ -44,29 +56,15 @@ export interface HomePageState {
...
+        text: "Projects",
     },
     {
-        href: "#message-anchor",
-        svgIcon: AnchorBotan,
-        text: "Messages",
-        threshold: .1,
+        href: "#messages-anchor",
+        svgIcon: ExpandMoreIcon,
+        text: "Projects",
```
Comment (R-Jim) [inline frontend/src/pages/home/home.tsx]: should it be `Messages`?

**Review by R-Jim — COMMENTED** [review]
lgtm

File: frontend/src/pages/home/home.tsx (hunk 1 above)
Comment (DuongTyler) [reply frontend/src/pages/home/home.tsx]: Ah yes, good catch

**Review by R-Jim — APPROVED** [review]

**Comment by DuongTyler** [comment]
Closes #3

**Comment by DuongTyler** [comment]
and closes #7

**Pull request merged**